docs/libcurl/curl_mprintf.3

```diff
@@ -47,57 +47,207 @@ curl_mvsprintf - formatted output conversion
 .br
 .BI "char *curl_mvaprintf(const char *" format ", va_list " args ");"
 .SH DESCRIPTION
-These are all functions that produce output according to a format string and
-given arguments. These are mostly clones of the well-known C-style functions
-and there will be no detailed explanation of all available formatting rules
-and usage here.
+These functions produce output according to the format string and given
+arguments. They are mostly clones of the well-known C-style functions but
+there are slight differences in behavior.
 
-See this table for notable exceptions.
-.RS
+We discourage users from using any of these functions in new applications.
```

Comment:
I think `deprecated` might be a better term as it is more common among various APIs

Reply:
deprecated implies it's going to be removed

Reply:
Leads to the same goal though

Reply:
I'm avoiding the use of word *deprecated* since it might give the impression that the function is going away in a foreseeable time. Some years ago I think we even anticipated that future, but these days I think we're more of the mindset that we will *never* remove any functions from the API (for all practical senses of the word, like decades).

I still want to discourage users from using the *mprintf() functions since they're not as complete and standards compliant like most modern *printf() implementations and it saves both users and us from problems if they rather lean on those than on our implementations.